docs/source/runner.md

```diff
@@ -210,3 +210,23 @@ class MyContainer(LightningContainer):
 To the optimizer and LR scheduler: the Lightning model returned by `create_model` should define its own
 `configure_optimizers` method, with the same signature as `LightningModule.configure_optimizers`,
 and returns a tuple containing the Optimizer and LRScheduler objects
+
+## Run inference with a pretrained model
+
+You can use the hi-ml-runner in inference mode only by switching the `--run_inference_only` flag on and specifying
+the model weights by setting any of those arguments `--weights_url`, `--local_weights_path` or `--ckpt_run_id`.
```

Comment:
Not sure I wrote that elsewhere: the naming does not look very consistent (weights/weights/ckpt). Also, how common is the abbreviation `ckpt`? `checkpoint_from_run` would be longer, but reads nicer. `checkpoint_url` and `local_checkpoint`?

Reply:
Sounds great, will rename them!